### PR TITLE
Specify that the changelog is not for docs

### PR DESCRIPTION
**Pull request by akarve**
# Description

# TODO

- [n/a] Unit tests
- [n/a] Automated tests (e.g. Preflight)
- [x] Documentation
    - [n/a] [Python: Run `build.py`](../gendocs/build.py) for new docstrings
    - [n/a] JavaScript: basic explanation and screenshot of new features
- [n/a] [Changelog](CHANGELOG.md) entry

### CODE REVIEW

**Review by smkohnstamm — APPROVED** [review]
Looks fine

**Review by smkohnstamm — APPROVED** [review]

**Review by smkohnstamm — APPROVED** [review]
All good.

File: docs/pull_request_template.md [hunk 1]
```diff
@@ -10,4 +10,4 @@
 - [ ] Documentation
     - [ ] [Python: Run `build.py`](../gendocs/build.py) for new docstrings
     - [ ] JavaScript: basic explanation and screenshot of new features
-- [ ] [Changelog](CHANGELOG.md) entry
+- [ ] [Changelog](CHANGELOG.md) entry (does not apply to documentation-only changes)
```
Comment (fiskus) [inline docs/pull_request_template.md:13]:
```suggestion
- [ ] [Changelog](CHANGELOG.md) entry (don't add entry if change is docs only)
```

More actionable wording?

**Review by fiskus — APPROVED** [review]

**Review by QuiltSimon — DISMISSED** [review]
Shorter and more succinct

File: docs/pull_request_template.md [hunk 2]
```diff
@@ -10,4 +10,4 @@
 - [ ] Documentation
     - [ ] [Python: Run `build.py`](../gendocs/build.py) for new docstrings
     - [ ] JavaScript: basic explanation and screenshot of new features
-- [ ] [Changelog](CHANGELOG.md) entry
+- [ ] [Changelog](CHANGELOG.md) entry (don't add entry if change is docs only)
```
Comment (sir-sigurd) [inline docs/pull_request_template.md:13]:
```suggestion
- [ ] [Changelog](CHANGELOG.md) entry (don't add entry if change is not significant to users e.g. docs only)
```

**Review by sir-sigurd — APPROVED** [review]

**Pull request merged**